Userland/Utilities/beep.cpp

```diff
@@ -4,11 +4,16 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include <LibCore/ArgsParser.h>
 #include <LibCore/System.h>
 #include <LibMain/Main.h>
 
-ErrorOr<int> serenity_main(Main::Arguments)
+ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
-    TRY(Core::System::beep());
+    Optional<size_t> tone;
+    Core::ArgsParser args_parser;
+    args_parser.add_option(tone, "Beep tone", "beep-tone", 'f', "Beep tone (frequency in Hz)");
+    args_parser.parse(arguments);
+    TRY(Core::System::beep(tone.value_or(440)));
```

Comment:
it looks sensible for me to have a default value in the syscall itself… 🤔 (i.e. i want a beep from the system; not knowing exactly which is the best—or even that there multiple possible—to think of it: the time should be also set! 😄)

Reply:
Yeah +1 for making `Core::System::beep()` default to 440.

Reply:
well… i had a `sys$beep` itself in mind – not only the core `beep`… 😟